packages/material-ui-types/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/types",
-  "version": "6.0.2",
+  "version": "6.0.3",
```

Comment:
Same as whatever we decide with https://github.com/mui-org/material-ui/pull/28093/files#r700189448